packages/cozy-urls/src/cozy-urls.js

```diff
@@ -0,0 +1,16 @@
+import { getSSL, getUrlAndHost, setAppName } from './helpers'
+
+const getProps = (cozyURL, sslOption) => {
```

Comment:
Default for `sslOption` should be `true` imho. What do you think ?

Reply:
We can't do that here since we won't be able to pass `null` or `undefined`